database/mock_data/PopulateDB_Part2.sql

```diff
@@ -31,3 +31,4 @@ left join membership ms on m.membership_id = ms.membership_id
 set membership_admin_id = m.member_id
 where m.member_type_id = 2;
 
+insert into board_member_title(title) VALUES ('TEST FOR DELETE');
```

Comment:
Can this line be in `Populate.sql` with the other `board_member_title` entries, just so they're all in the same place?